src/node_crypto.cc

```diff
@@ -25,6 +25,7 @@
 #include "node_crypto.h"
 #include "node_crypto_bio.h"
 #include "node_crypto_groups.h"
+#include "node_mutex.h"
```

Comment:
The file needs this `#include`. It was working because [`debug-agent.h`](https://github.com/nodejs/node/blob/37699070cc3aea7c2a0b8b764c8314a301e977fa/src/debug-agent.h#L27) had it.